File: Assignments/Assignment_01/Task_07.java
    public static boolean splittingArray(int[] source){
        for(int i=1, j=source.length-1,s1=0,s2=0; i<source.length-1; i++){
            for (int k=0;k<i;k++){
                s1 += source[k];
            }
            for (int x=j; x>=i; x--){
                s2 += source[x];
            }
            if (s1==s2){
                return true;
            }
            s1 = 0;
            s2 = 0;
        }
        return false;
    }

File: Assignments/Assignment_01/Spring-23/Task_07.py
# Test 07: Splitting an Array
def split_array(source):

    for i in range(1,len(source)):
        j = len(source)-1
        s1 = 0
        s2 = 0
        for k in range(i):
          s1+=source[k]
        for x in range(j,i-1,-1):
          s2+= source[x]
        if s1 == s2:
          return True
        s1,s2 = 0,0